docs/developer-guide/ksqldb-clients/contributing.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: Contributing a new ksqlDB client
+tagline: Contributing a new ksqlDB client
+description: Contribute a new ksqlDB client for your favorite language.
+keywords: ksqlDB, client
+---
+
+This page contains information and recommendations for how to contribute a new [ksqlDB client](index.md).
```

Comment:
Maybe we could say something more inviting, like: "If you're interested in using ksqlDB from a programming language we don't yet support, we've built out a guide to make it easy to add your own client. .... "